core/example/benchmark/CMakeLists.txt

```diff
@@ -1,7 +1,15 @@
+option(Cabana_ENABLE_EXAMPLES "Build examples" OFF)
```

Comment:
You should define the option in main CMake, it is double defined below!

Reply:
Thanks, that was a copy paste error I'll take it out!